### PR TITLE
Adding some additional clarification for mentees.

### PR DESCRIPTION
**Pull request by jkowall**
Based on some of the questions we are getting from our new Mentees we wanted to add some additional pointers and clarification ot make it easier for them to get started. 

Added link for the current ongoing LFX mentorship. 

## Checklist
- [X] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [X] I have signed all commits

Tested new page locally.

### CODE REVIEW

File: content/mentorship-for-mentees.md [hunk 1]
```diff
@@ -29,7 +29,12 @@ It does not mean that the longer the proposal the better. It's about the quality
 In order to understand the project better and come up with reasonable solutions, it's always helpful to become familiar with Jaeger and its code base. We strongly recommend:
 
 * Going through some Jaeger tutorials, such as [this one][hotrod].
-* Trying to solve some of the [simple open issues](../get-involved/) that you can find across Jaeger repositories.
+* Review the hotrod video demo and tutorial such as [this video on youtube][hotrod_video].
```
Comment (yurishkuro) [inline content/mentorship-for-mentees.md:32]:
```suggestion
* Review the HotROD video demo and tutorial such as [this video on youtube][hotrod_video].
```
HotROD ™️

File: content/mentorship-for-mentees.md [hunk 2]
```diff
@@ -29,7 +29,12 @@ It does not mean that the longer the proposal the better. It's about the quality
...
+* Run the [hotrod demo yourself][hotrod_repo] as the blogs and video is outdated, it's good to get hands on.
+* Review the Jaeger architecture from the [documentation](https://www.jaegertracing.io/docs/architecture/) and understand the microservices architecture.
```
Comment (yurishkuro) [inline content/mentorship-for-mentees.md:34]: > understand the microservices architecture.

what do you mean by that?

File: content/mentorship-for-mentees.md [hunk 3]
```diff
@@ -29,7 +29,12 @@ It does not mean that the longer the proposal the better. It's about the quality
 In order to understand the project better and come up with reasonable solutions, it's always helpful to become familiar with Jaeger and its code base. We strongly recommend:
 
 * Going through some Jaeger tutorials, such as [this one][hotrod].
```
Comment (yurishkuro) [inline content/mentorship-for-mentees.md:31]:
```suggestion
* Going through some Jaeger tutorials, such as [HotROD demo][hotrod] or [YouTube video][hotrod_video]
```

File: content/mentorship-for-mentees.md [hunk 4]
```diff
@@ -29,7 +29,12 @@ It does not mean that the longer the proposal the better. It's about the quality
...
-* Trying to solve some of the [simple open issues](../get-involved/) that you can find across Jaeger repositories.
+* Review the hotrod video demo and tutorial such as [this video on youtube][hotrod_video].
+* Run the [hotrod demo yourself][hotrod_repo] as the blogs and video is outdated, it's good to get hands on.
```
Comment (yurishkuro) [inline content/mentorship-for-mentees.md:33]:
```suggestion
* Run the [HotROD demo yourself][hotrod_repo] as the blogs and video is outdated, it's good to get hands on.
```
HotROD ™️

File: content/mentorship-for-mentees.md (hunk 2 above)
Comment (yurishkuro) [inline content/mentorship-for-mentees.md:34]:
```suggestion
* Review the [Jaeger architecture documentation](https://www.jaegertracing.io/docs/architecture/) and understand the microservices architecture.
```

> understand the microservices architecture.

what do you mean by that?

Comment (yurishkuro) [reply content/mentorship-for-mentees.md:34]: > https://www.jaegertracing.io/docs/architecture/

better to use relative path

Comment (jkowall) [reply content/mentorship-for-mentees.md:34]: Understanding of the components in the architecture is another way to phrase it, if you prefer.

I will edit to use relative on the push.

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**